src/cmd-buildextend-live

```diff
@@ -294,18 +294,33 @@ def generate_iso():
     with open(os.path.join(tmpisocoreos, 'features.json'), 'w') as fh:
         fh.write(features)
 
+    # Get PRETTY_NAME
+    with tempfile.TemporaryDirectory() as tmpd:
+        run_verbose(['/usr/bin/ostree', 'checkout', '--repo', repo,
+                     '--user-mode', '--subpath', "/usr/lib/os-release",
+                     buildmeta_commit, tmpd])
+        pretty_name = subprocess.check_output(['sh', '-euc', '. ./os-release; echo -n $PRETTY_NAME'],
+                                              encoding='utf-8', cwd=tmpd)
+        # escaped version that survives runvm; yuck
+        pretty_name_esc = f"'{pretty_name}'"
```

Comment:
Probably fine as is, but in the Go future we should use go-shellquote.

(I guess as is now we could `from gi.repository import GLib; GLib.shell_quote`)

Hmm actually though shouldn't the quoting responsibility be in `runvm`?